### PR TITLE
Moving to Python3

### PR DESCRIPTION
**Pull request by iambibhas**
Related to https://github.com/hasgeek/hgapp/issues/39

### CODE REVIEW

File: boxoffice/views/admin_item.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 from boxoffice.models import db, Organization, ItemCollection, Item, Price
 from boxoffice.views.utils import api_error, api_success, json_date_format
 from boxoffice.forms import ItemForm, PriceForm
```
Comment (jace) [inline boxoffice/views/admin_item.py]: These imports have to change to `..`

File: boxoffice/views/admin_order.py [hunk 2]
```diff
@@ -4,7 +4,7 @@
 from .. import app, lastuser
 from coaster.views import load_models, render_with
 from boxoffice.models import ItemCollection, Organization, Order, CURRENCY_SYMBOL, LineItem, LINE_ITEM_STATUS, ORDER_STATUS, INVOICE_STATUS
```
Comment (jace) [inline boxoffice/views/admin_order.py]: Could you correct `from boxoffice` imports across all files?

File: scripts/sync_razorpay_refunds.py [hunk 3]
```diff
@@ -45,7 +45,7 @@ def sync_refunds():
                 correspongding_rp_refund = min(possible_rp_refunds,
                     key=lambda rp_refund: abs(rp_refund['created_at'] - refund_epoch_dt))
                 if not correspongding_rp_refund or amount_in_paise(refund.amount) != correspongding_rp_refund['amount']:
-                    raise("Oops! No refund found for {refundid}.".format(refundid=refund.id))
+                    raise "Oops! No refund found for {refundid}."
```
Comment (jace) [inline scripts/sync_razorpay_refunds.py]: The format parameter is missing.

File: .travis.yml [hunk 4]
```diff
@@ -12,7 +12,6 @@ install:
   - pip install -U pip wheel
   - pip install -r test_requirements.txt
   - pip install -r requirements.txt
-  - pip install https://github.com/hasgeek/coaster/archive/master.zip
```
Comment (jace) [inline .travis.yml]: Doesn't this work if it's in `requirements.txt`?

Comment (iambibhas) [reply .travis.yml]: It should. I'm not sure why it was added here.

**Pull request merged**